task60_PageObject/src/test/java/tut/by/pages/HomePage.java

```diff
@@ -0,0 +1,34 @@
+package tut.by.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+
+import static org.testng.Assert.assertTrue;
+
+public class HomePage  extends BasePage {
+    private By LOGIN_WINDOW_BUTTON = By.className("enter");
+    private By USER_NAME = By.className("uname");
+    private By LOGOUT_BUTTON = By.className("auth__reg");
+
+    public HomePage(WebDriver driver){
+       super(driver);
+       assertTrue(isDisplayed(LOGIN_WINDOW_BUTTON));
```

Comment:
Please avoid assertions in page objects. I recommend to throw exception here, if element is not displayed

Reply:
I've added try-catch to the "getElement" method in BasePage class